gulpfile.js

```diff
@@ -105,7 +105,7 @@ var buildScripts = function (done) {
 	if (!settings.polyfills) {
 		scriptSrc.push('!' + paths.scripts.polyfills);
 	}
-	return src(scriptSrc)
+	return src(scriptSrc, { since: lastRun(buildScripts) })
```

Comment:
🔍 `lastRun` allows Gulp to only source files that have changed since the last run of the provided function. In this case, `buildScripts` will only minify and copy .js files that have changed since last build.